dist/angular-local-storage.js

```diff
@@ -136,7 +135,13 @@ angularLocalStorage.provider('localStorageService', function() {
       }
     }());
 
-
+    // Reviver function for JSON.parse that will be called
+    // for every key and value at every level of the final string -> JSON transformation
+    var reviver = function(key, value) {
+      if (value === 'true' || value === 'false') { return (value === 'true'); }
```

Comment:
- Please use function declaration instead.
- There's no point to wrap it with `{}`.
- Also, these parentheses unnecessary here.